packages/idyll-components/src/feature.js

```diff
@@ -64,12 +66,49 @@ class Feature extends React.PureComponent {
     window.addEventListener('scroll', this.handleScroll.bind(this));
   }
 
+  unwrapChild(c) {
+    if (c => c.type.name && c.type.name.toLowerCase() === 'wrapper') {
+      return c.props.children[0];
+    }
+    return c;
+  }
+
+  unwrapChildren() {
+    return this.props.children.map((c) => this.unwrapChild(c));
+  }
+
+  getFeatureChild() {
+    const unwrapped = this.unwrapChildren();
+    const filtered = React.Children.toArray(this.props.children).filter((_, i) => {
+      const c = unwrapped[i];
+      if (!c.type) {
+        return false;
+      }
+      return (c.type.name && c.type.name.toLowerCase() === 'content') || c.type.prototype instanceof Content;
+    })
+    if (filtered.length) {
+      return filtered[0];
+    }
+  }
+
+  getNonFeatureChildren() {
+    const unwrapped = this.unwrapChildren();
+    return React.Children.toArray(this.props.children).filter((_, i) => {
+      const c = unwrapped[i];
+      if (!c.type) {
+        return true;
+      }
+      return (!c.type.name || c.type.name.toLowerCase() !== 'content') && !(c.type.prototype instanceof Content);
+    });
+  }
+
```

Comment:
Nit, but combining these functions would be nice. Could return an array or object perfect for destructuring.